packages/grafana-data/src/dataframe/DataFrameView.ts

```diff
@@ -48,6 +49,12 @@ export class DataFrameView<T = any> implements Vector<T> {
     this.obj = obj;
   }
 
+  *[Symbol.iterator]() {
+    for (let i = 0; i < this.data.length; i++) {
+      yield { ...this.get(i) };
```

Comment:
AFAIK using the spread operator can sometimes have performance penalties versus, say, using a for loop to clone an object, but I haven't run any benchmarks to see if it would be an issue here.